README.md

```diff
@@ -116,6 +117,19 @@ An array of package names needed for httpd Phusion Passenger support.  The defau
 
 This class manages the Apache web server to provide HTTPS support.
 
+
+#### apache::mod\_status class
+
+This class manages the Apache status module which provides reports of server activity.  By default access will
+only be allowed on the local loopback interface.
+
+##### `location`
+This parameter controls the location used to access the apache status page.  Default value is set to /server-status.
```

Comment:
Can you please change this so that default is represented as code, specifically a quoted string.  That way it'll be consistent, see one of the *ensure* parameters.